<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QMainWindow" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>chat</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Images/chat.png</normaloff>../../../Images/chat.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pb_newConver">
        <property name="text">
         <string>nouvelle conversation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::West</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="le_message"/>
        </item>
        <item>
         <widget class="QPushButton" name="pb_envoyer">
          <property name="text">
           <string>Envoyer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_pdf">
          <property name="text">
           <string>Export PDF</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
